src/objc/ModalWebViewController.mm

```diff
@@ -84,27 +84,46 @@ - (void)viewDidDisappear:(BOOL)animated {
   }
 }
 
-- (void)getBrowserCookiesForCurrentUrlWithCompletion:(void (^)(NSDictionary *))completion {
+- (void)getBrowserCookiesForUrlWithCompletion:(NSString *)domain completion:(void (^)(NSDictionary *))completion {
```

Comment:
The name doesn't match anymore, this is not a url but a raw string comparison, which might be missleading as one can pass even non-valid urls